phpstan-baseline.neon

```diff
@@ -0,0 +1,12 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Class JMS\\\\Serializer\\\\Handler\\\\iterable not found\\.$#"
```

Comment:
hmm, this is weird... why do we get this error?

Reply:
I don't know yet. Could not reproduce it on phpstan playground.

Reply:
maybe it needs just a `\`?

Reply:
it's not an object so cannot be imported. It's like `string`.

https://phpstan.org/r/84508559-53e5-43da-b083-219d3b9af0a5

I was thinking that would investigate it later, that's why I added it to baseline for now.

Reply:
I found the problem, see https://github.com/schmittjoh/serializer/pull/1315